common/src/main/java/com/microsoft/identity/common/model_old/Account.java

```diff
@@ -1,4 +1,4 @@
-package com.microsoft.identity.common;
+package com.microsoft.identity.common.model_old;
```

Comment:
@shoatman Presumably this class goes away/gets refactored into the new implementation?

Reply:
The account object that we originally defined has behavior and responsibilities.  The Account object as defined by the new cache schema is a DTO.  We may want to keep these separate.  Thoughts?

Reply:
I don't have any objection to the _role_ of this object, though the naming might lead to some confusion -- do we consider renaming?